rules-reviewed/technology-usage/tests/samples.http/JaxWsWebService.java

```diff
@@ -0,0 +1,43 @@
+package org.windup.examples.technology.http;
```

Comment:
Same as above.

Reply:
I don't understand? In #262 there is `package org.windup.examples.ejb.messagedriven;` so based on that, I Used `package org.windup.examples.technology.http`. What is wrong on that?

https://github.com/jsight/windup-rulesets/blob/b39a22f676f882d4c202af4d73e03c663246cb59/rules-reviewed/technology-usage/tests/data/messaging/MessageDrivenBean2.java

Reply:
It was related to have test data in `tests/data/http/`